src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
+
+  &__card {
+    display: flex;
```

Comment:
I'm not sure, but it seems to me that a grid should also be used here

Reply:
no, for card you can still use flex)